worker-default-configs/config/cfg~caf~worker~FileSystemDataStoreConfiguration.js

```diff
@@ -15,6 +15,8 @@
  */
 ({
     dataDir: getenv("CAF_WORKER_DATASTORE_PATH") || "/mnt/caf-datastore-root",
+    dataDirHealthcheckTimeoutSeconds: getenv("CAF_WORKER_DATASTORE_HEALTHCHECK_TIMEOUT_SECONDS")
+            || 10,
```

Comment:
My advice w.r.t. this btw is to fallback to `undefined` the same way `outputBufferSize` is doing, and then to put the default in the code, again like [this](https://github.com/WorkerFramework/worker-framework/blob/v3.3.0/worker-store-fs/src/main/java/com/hpe/caf/worker/datastore/fs/FileSystemDataStore.java#L348).  It will save you having to worry about components which don't use the default configuration file.